### PR TITLE
OpenAPI update

### PR DESCRIPTION
**Pull request by Majkluss**

### CODE REVIEW

File: spec/openapi.yaml [hunk 1]
```diff
@@ -112,30 +117,35 @@ paths:
...
+                      example: "connect.prusa3d.com"
                     port:
                       type: integer
-                      default: 8080
+                      default: 0
                     tls:
-                      type: boolean
-                      default: false
+                      type: integer
+                      default: 1
```
Comment (Lowentwickler) [inline spec/openapi.yaml:148]: why don't you want to have it as a boolean?

File: spec/openapi.yaml [hunk 2]
```diff
@@ -198,6 +208,7 @@ paths:
                     $ref: "#/components/schemas/PrinterState"
                   telemetry:
                     oneOf:
+                      - $ref: "#/components/schemas/TelemetryMK3"
```
Comment (Lowentwickler) [inline spec/openapi.yaml:211]: You sure?

**Comment by Majkluss** [comment]
Deprecated - new OpenAPI is in progress

File: spec/openapi.yaml (hunk 1 above)
Comment (Majkluss) [reply spec/openapi.yaml:148]: Fixed back to boolean

File: spec/openapi.yaml (hunk 2 above)
Comment (Majkluss) [reply spec/openapi.yaml:211]: Fixed

**Review by Lowentwickler — APPROVED** [review]

File: spec/openapi-legacy.yaml [hunk 3]
```diff
@@ -677,12 +717,156 @@ paths:
...
+                        example: "/api/files/local/Prusa Link gcodes/castle.gcode"
+                      download:
+                        type: string
+                        example: "/api/files/local/Prusa Link gcodes/castle.gcode/raw"
+                      thumbnailSmall:
+                        type: string
+                        example: null
+                      thumbnailBig:
+                        type: string
+                        example: "/api/thumbnails/Prusa Link gcodes/castle.gcode.orig.png"
```
[the remark this answers is not in the document]
Comment (Majkluss) [reply spec/openapi-legacy.yaml:785]: No, it's correct endpoint for thumbnail in original size

**Pull request merged**